apps/trading/client-pages/transfer/transfer.spec.tsx

```diff
@@ -0,0 +1,70 @@
+import { act, render, screen, waitFor } from '@testing-library/react';
+import { MockedProvider } from '@apollo/react-testing';
+import { MemoryRouter } from 'react-router-dom';
+import { Transfer } from './transfer';
+import { useTransferAssetIdStore } from '@vegaprotocol/accounts';
+import type { VegaWalletContextShape } from '@vegaprotocol/wallet';
+import { VegaWalletContext } from '@vegaprotocol/wallet';
+
+const mockWalletContext = {} as unknown as VegaWalletContextShape;
+
+let mockSearchParamsResult: string | undefined = undefined;
+const mockSearchParams = {
+  get: jest.fn(() => mockSearchParamsResult),
+};
+const mockSetSearchParams = jest.fn();
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useSearchParams: jest.fn(() => [mockSearchParams, mockSetSearchParams]),
+}));
+jest.mock('@vegaprotocol/accounts', () => ({
+  ...jest.requireActual('@vegaprotocol/accounts'),
+  TransferContainer: jest.fn(({ assetId }: { assetId?: string }) => (
+    <div data-testid="assetId">{assetId}</div>
+  )),
+}));
+
+const renderJsx = () => {
+  render(
+    <VegaWalletContext.Provider value={mockWalletContext}>
```

Comment:
This is not needed as the container is mocked

Reply:
Actually it was needed for the GetStarted component, but I think it simpler to mock it. I've pushed changes